### DISCUSSION TITLE
A question about randomly splitting the Planetoid datasets

### DISCUSSION THREAD

**Discussion by OliverSummers** [pyg-team/pytorch_geometric#3040] [Q&A] [answered]
How many nodes should we choose for training, validating, and testing when _randomly_ splitting the Planetoid datasets, i.e. Cora, Citeseer and PubMed? I am asking this since if I use 50 as the number of nodes from each class of Cora (all in total 350 nodes for training), I get higher accuracy (around 5% higher) than the case in which I use 20 as the number of nodes from each class (all in total 140 nodes for training).

**Comment by rusty1s** [comment, marked as answer]
More training labels usually results in better model performance. As such, there is no rule how many nodes should be used for training (and usually depends on how many labels are given overall). If you want to compare against existing methods, you need to take care of evaluating on the same split ratios to make the comparison fair.

**Reply by OliverSummers** [reply]
Thank you.
